spec/factories/moods.rb

```diff
@@ -1,5 +1,6 @@
 FactoryBot.define do
   factory :mood do
     name { Faker::SlackEmoji.people }
+    rating { Faker::Number.between(from: 1, to: 3) }
   end
 end
```

Comment:
The way this factory is set up still doesn't seem like an accurate portrayal of a mood but I am not sure if that matters.

Reply:
I'm using it primarily for model testing such as the User spec where the mood information doesn't necessarily matter. I don't use it for the actual mood testing where we need specific information such as rating and mood names